### ISSUE TITLE
Workorder script (for a remedy) has failed match with low-quality item

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#1483]
The script recovers and continues the workorder, but it the low-quality items are taking up a lot of spac e in my backpack.

```
2017-03-15 09:51:09 -0400:[remedy]>get my salve from my mortar
2017-03-15 09:51:09 -0400:You get some nilos salve from inside your iron mortar.
2017-03-15 09:51:09 -0400:[remedy]>put my iron mortar in my pack
2017-03-15 09:51:09 -0400:You put your mortar in your leather pack.
2017-03-15 09:51:09 -0400:--- Lich: remedy has exited.
2017-03-15 09:51:10 -0400:[workorders]>get my alchemy logbook
2017-03-15 09:51:10 -0400:You get an alchemy work order logbook from inside your leather pack.
2017-03-15 09:51:10 -0400:[workorders]>bun my salve with logbook
2017-03-15 09:51:10 -0400:The work order requires items of a higher quality, so you decide against bundling that.
2017-03-15 09:51:10 -0400:[workorders]>get my salve
2017-03-15 09:51:10 -0400:You are already holding that.
2017-03-15 09:51:25 -0400:[workorders: *** No match was found after 15 seconds, dumping info]
2017-03-15 09:51:25 -0400:[workorders: messages seen length: 16]
2017-03-15 09:51:25 -0400:[workorders: message: You are already holding that.]
2017-03-15 09:51:25 -0400:[workorders: checked against [/You get/i, /What were you/i]]
2017-03-15 09:51:25 -0400:[workorders]>put my nilos salve in my pack
2017-03-15 09:51:26 -0400:You put your salve in your leather pack.
2017-03-15 09:51:26 -0400:[workorders]>put my alchemy logbook in my pack
2017-03-15 09:51:26 -0400:You put your logbook in your leather pack.
```

**Comment by Dartellum** [comment]
I think I fixed this one as well. I added an exit when the failure occurs. However, after 15 runs on hard, I have not failed a recipe. I will keep running.

**Comment by rpherbig** [comment]
Fixed in a5dfd841da50f688fef0f3d6d1bd029c5fbca40e
